test/SolverInterface.cpp

```diff
@@ -350,6 +353,20 @@ void SolverInterface:: readScalarData
     value = fake_read_write_buffer[0]; 
 }
 
+void SolverInterface:: setMeshAccessRegion
+(
+  int meshID,
+  const double* boundingBox ) const
+{}
+
+void SolverInterface:: getMeshVerticesAndIDs
+(
+  int meshID,
+  int size,
+  int* valueIndices,
+  double* coordinates ) const
+{}
```

Comment:
Here, you could generate some dummy values on the cpp side and check whether you receive them correctly on the python side.